rasa/core/training/story_reader/markdown_story_reader.py

```diff
@@ -233,7 +233,7 @@ def is_markdown_story_file(file_path: Text) -> bool:
         """
         suffix = PurePath(file_path).suffix
 
-        if suffix and suffix != MARKDOWN_FILE_EXTENSION:
+        if suffix not in MARKDOWN_FILE_EXTENSIONS:
```

Comment:
I think we should discuss if we keep the original behaviour: I changed it to keep it uniform across readers. The difference between the versions is that the original one would happily read files without an extension as story files, which seems arbitrary. Nevertheless, it is breaking backwards compatibility in an odd way, so kind of betting that no one relies on this odd behaviour 🤔

Reply:
Should we add this to the changelog? `Training data form in markdown format` has to have the file suffix `.md` from now on`?